pkg/driver/docker/docker.go

```diff
@@ -284,10 +296,31 @@ func (c dockerClient) StartBuildkitd(ctx context.Context, tag, name string, bc *
 	}
 	created, _ := c.Exists(ctx, name)
 	if created {
-		err := c.ContainerStart(ctx, name, types.ContainerStartOptions{})
+		status, err := c.GetStatus(ctx, name)
 		if err != nil {
-			return name, err
+			return name, errors.Wrap(err, "failed to get container status")
+		}
+
+		if status == StatusPaused {
+			logger.Info("container was paused, unpause it now...")
+			_, err := c.ResumeContainer(ctx, name)
+			if err != nil {
+				return name, errors.Wrap(err, "failed to unpause container")
+			}
+		} else if status == StatusExited || status == StatusDead || status == StatusRemoving {
```

Comment:
`removing` and `dead` cannot be restarted. The long if-else should be extracted as a function.

Reply:
Should I judge `removing` and `dead` condition? Or just remove them?